### ISSUE TITLE
E-Field not correctly moving with the moving window

### ISSUE THREAD

**Issue by TimArniko** [SmileiPIC/Smilei#274]
Last year, the bug of the moving window related to the ion energy spectra has been fixed.
I wonder, if there is an other bug of the moving window, related to the E-field this time.

The description of the simulation: accelerating ions with a constant 2D-laser shape (laser driven radiation pressure acceleration of ions).

Normally, the tconstant()-function should produce a constant laser for all times. Actually, I think, that the laser stops beeing produced when the moving window is starting. I have for example this parameters in my code:

```
MovingWindow(
    time_start = 33.*t0,
    velocity_x = 0.67,
)

LaserPlanar1D(
              box_side         = "xmin",
              a0               = 150.,
              omega            = 1.,
              polarization_phi = 0.,
              ellipticity      = 1.,
              time_envelope    = tconstant()
)
```

but other parameters induce the same problem.

I attached also a figure where we can see the stopping of the laser field after the moving window shifted the simulation to the left:
![E_perp_vs_time](https://user-images.githubusercontent.com/67972379/86824583-9ec7ba80-c08e-11ea-8572-a24e0128e0f2.png)

**Comment by mccoys** [comment]
This is not a bug. The moving window cancels the laser injection because it would require complex recalculation of the fields at different positions while the window moves. It is true that the documentation should explain this, so I am changing the tags of this issue.

Do you actually know a code which can do that ?

**Comment by mccoys** [comment]
@beck-llr I believe the doc has been fixed ?

**Comment by beck-llr** [comment]
Yes it has. There is now a warning: "When the window starts moving, all laser injections via Silver-Muller boundary conditions are immediately stopped for physical correctness."
